sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -115,6 +115,8 @@ Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to creat
 
 ## Authenticating the client
 
+This library supports the use of [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) which provides a variety of credential types that your application can use to authenticate users and services with the Key Vault service.  For more information on how to use these credentials, read [Using @azure/identity with Microsoft Identity Platform](/documentation/using-azure-identity.md).
```

Comment:
I would point to a link to the identity readme before the using-azure-identity.md. The second perhaps goes too deep to fast. This same feedback would apply to all the readmes that were updated with this blurb

Reply:
Good point!